config/authorization/config.ex

```diff
@@ -127,6 +163,35 @@ defmodule Acl.UserGroups.Config do
                         "http://www.w3.org/ns/prov#Location"
                       ]
                     } } ] },
+      # // Admin users
+      %GroupSpec{
+        name: "o-admin-sessions-rwf",
+        useage: [:read, :write, :read_for_write],
+        access: is_admin(),
+        graphs: [
+          %GraphSpec{
+            graph: "http://mu.semte.ch/graphs/sessions",
+            constraint: %ResourceFormatConstraint{
+              resource_prefix: "http://mu.semte.ch/sessions/"
+            }
+          },
+        ]
+      },
+      %GroupSpec{
+        name: "o-admin-sessions-rwf",
```

Comment:
@Windvis @Riadabd Key was using the `:read_for_write:` flag for public graph and admin users. This allows admin users to read from a graph for insert/delete statement, even if the admin user can't write to it. Exactly what's happening here: https://github.com/lblod/impersonation-service/blob/master/lib/session.js#L99